src/main/java/seedu/planner/model/ModulePlanner.java

```diff
@@ -448,6 +441,22 @@ private void sortAvailableModules(List<Module> modulesAvailable, UserProfile use
         return MajorDescription.getModuleCode(userProfile.getMajor(), code);
     }
 
+    /**
+     * Count the number of general education modules
+     */
+    private void countGeneralEducation() {
+        int count = 0;
+        for (Module m : getAllModulesTaken()) {
+            if (m.toString().startsWith("GE")) {
```

Comment:
Unfortunately, you can't just test `GE` prefix to determine if a module is under General Education. `GE` (two letters) is the module prefix for Geography major's modules too.

Reply:
Done